doc/refman/RefMan-pre.tex

```diff
@@ -1335,6 +1335,157 @@ \section*{Credits: version 8.7}
 Matthieu Sozeau and the {\Coq} development team\\
 \end{flushright}
 
+\section*{Credits: version 8.8}
+
+{\Coq} version 8.8 contains the result of refinements and stabilization of
+features and deprecations, cleanups of the internals of the system along
+with a few new features. The main user visible changes are:
+\begin{itemize}
+\item Kernel: fix a subject reduction failure due to allowing fixpoints
+  on non-recursive values, which allows to recover full parametricity
+  for CIC, by Matthieu Sozeau. Handling of evars in the VM (the kernel
+  still does not accept evars) and bugfix in the handling of universes
+  in the VM (backported to 8.7 as well) by Maxime Dénès.
```

Comment:
The VM fix is in a previous version, so you should remove it from here.

Reply:
ok